packages/perseus-editor/src/article-editor.tsx

```diff
@@ -350,7 +362,7 @@ export default class ArticleEditor extends React.Component<Props, State> {
         const newSection =
             i >= 0
                 ? {
-                      widgets: sections[i].widgets,
+                      widgets: sections![i].widgets,
```

Comment:
Shouldn't this be a question mark instead of an exclamation point?

Reply:
I'm fixing this one by getting rid if the silly local type that's faking the PerseusRenderer.